src/sparseml/transformers/export.py

```diff
@@ -246,9 +246,10 @@ def export_transformer_to_onnx(
         )
 
     if num_export_samples > 0 and data_args is None:
-        raise ValueError(
+        _LOGGER.info(
             f"--data_args is needed for exporting {num_export_samples} "
-            f"samples but got {data_args}"
+            f"real samples but got {data_args}, fake samples will be generated "
```

Comment:
nit, but personally not a fan of print {variable} when you're catching cases where variable is empty. I think it's better to explicitly state that the arg was None or wasn't provided